dnsserv_test.go

```diff
@@ -1,40 +0,0 @@
-package main
-
-import (
-	"bytes"
-	. "gopkg.in/check.v1"
-	"net"
-	"testing"
-)
-
-func Test(t *testing.T) { TestingT(t) }
-
-type DNSNameserverSuite struct{}
-
-var _ = Suite(&DNSNameserverSuite{})
-
-func (s *DNSNameserverSuite) TestUDPConnection(c *C) {
```

Comment:
I completely agree with Y and R here - having a test that just checks the ip and port is very marginal value - and the time spent arguing on this could by itself have been spent better, I think. It is definitely a good idea not testing things where we are not responsible for the code. Further - adding unit tests has a cost, not just a benefit.